interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -276,10 +278,10 @@ namespace cling {
     }
 
     if(m_Opts.CompilerOpts.CUDAHost){
-       if(m_DyLibManager->loadLibrary("libcudart.so", true) ==
-         cling::DynamicLibraryManager::LoadLibResult::kLoadLibNotFound){
-           llvm::errs() << "Error: libcudart.so not found!\n" <<
-             "Please add the cuda lib path to LD_LIBRARY_PATH or set it via -L argument.\n";
+      if (getDynamicLibraryManager()->loadLibrary("libcudart.so", true) ==
```

Comment:
`libcudart.so`: is this code supposed to work also on Windows? (just wondering)

Reply:
Googling does not give me conclusive information... @SimeonEhrig or @ioanaif might know what to do there.

Reply:
No, the code should not work under Windows. I think it have to search for the `cudart.lib` on Windows. But I don't believe, that the CUDA mode is running on windows at the moment. But I never tested it.